lib/services/sf.js

```diff
@@ -1481,7 +1481,11 @@ StateRenewing.prototype.enter = function (context)
       json:
         {
           'REQUEST_TYPE': 'RENEW',
-          'oldSessionToken': this.tokenInfo.getSessionToken()
+          'oldSessionToken': this.tokenInfo.getSessionToken(),
+          data: {
+            CLIENT_APP_ID: this.connectionConfig.getClientType(),
```

Comment:
Do we really need it? There is no "data" section in [the spec](https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/54690803/Developer+REST+APIs#DeveloperRESTAPIs-POST%2Fsession%2Ftoken-request).  Other drivers (eg. .NET) don't fill "data" in token-request.

Reply:
Or maybe you wanted to add in the request header?